ArgentPonyWarcraftClient.sln

```diff
@@ -11,6 +11,10 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{9BBC9F4C-0E0
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "tests", "tests", "{BED6344B-B8DB-457B-8106-E8CEF5F42570}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ArgentPonyWarcraftClient.Extensions.DependencyInjection", "src\ArgentPonyWarcraftClient.Extensions.DependencyInjection\ArgentPonyWarcraftClient.Extensions.DependencyInjection.csproj", "{95EFCE5F-F1A4-49CF-874E-0B7682A8FB3C}"
```

Comment:
Minor: It would probably be better to use `{9A19103F-16F7-4668-BE54-9A1E7A4F7556}` instead of `{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}` as the project GUID.  See [this comment](https://github.com/dotnet/project-system/issues/3079#issuecomment-354180353) for details.  This is not a significant issue, and it could be addressed as a follow-up change.

Reply:
Huh, I had no idea about this! I'll have to look into it to understand it a little more and how the project system really works. I'll add the change though it's a pretty trivial fix. Better to take care of a small, simple fix now as opposed to putting it off 😄